lib/OpenQA/WebAPI/Auth/OpenID.pm

```diff
@@ -82,7 +82,6 @@ sub auth_login {
 sub auth_response {
     my ($self) = @_;
 
-    # FIXME: Mojo6 hack, remove after version bump
```

Comment:
I understood that the following section is assumed to be not necessary anymore as soon as there would be Mojolicious7 which might never happen and we don't know if it will fix it. FIXME sounds to me like "Please, fix me", but there is nothing we can do here so I removed the comment completely. What would you propose?